Blish HUD/Controls/Intern/Keyboard.cs

```diff
@@ -74,17 +41,21 @@ public void Press(GuildWarsControls key)
             }
             else
             {
-                uint vkCode = (uint)VirtualKeyShorts[key];
+                uint vkCode = (uint)key;
                 ExtraKeyInfo lParam = new ExtraKeyInfo(){
                     scanCode = (char)PInvoke.MapVirtualKey(vkCode, MAPVK_VK_TO_VSC)
                 };
-                PInvoke.PostMessage(Gw2WndHandle, WM_KEYDOWN, vkCode, lParam.GetInt());
+                PInvoke.PostMessage(GameService.GameIntegration.Gw2Process.MainWindowHandle, WM_KEYDOWN, vkCode, lParam.GetInt());
```

Comment:
Use `GameService.GameIntegration.Gw2WindowHandle` instead of pulling it from the `Gw2Process` since it is already exposed.

Reply:
Reviewed and it looks like this was fixed.  Also thanks for updating the scope.